presidio-anonymizer/presidio_anonymizer/anonymizers/replace.py

```diff
@@ -1,15 +1,12 @@
 """Replaces the PII text entity with new string."""
-from anonymizers.anonymizer import Anonymizer
+from presidio_anonymizer.anonymizers.anonymizer import Anonymizer
 
 
 # TODO implement + test
 class Replace(Anonymizer):
     """Receives new text to replace old PII text entity with."""
 
-    def __init__(self,
-                 new_text: str):
-        self.new_text = new_text
-
-    def anonymize(self):
-        """:return: new_text."""
-        pass
+    def anonymize(self, params={}):
+        """:return: new_val."""
+        new_val = params.get("new_value")
```

Comment:
I suggest not to leave implementations like that.
If it's not implemented, it's better return nothing, or throw not implemented exception.
The doc string is also not relevant and should be removed for now.

Reply:
Each class like this is marked with a TODO of test and implement. 
I know usually we need to put an exception but I need this to pass the tests otherwise the flow won't work.
Also, there are tickets for it as well

Reply:
Please see the last line of my comment about the doc string.